CONTRIBUTING.md

```diff
@@ -43,15 +43,23 @@ we recommend the following workflow:
 
 1. Optional but recommended to prevent complaints from our CI pipeline: Test your code.
 
-   Testing against a single python can be achieved via `tox -e <version>`. For instance:
-   ```bash
-   tox -e py311
```

Comment:
@AdrianSosic I've pushed a correction to this, there is no environment defined like `py311` and it will do nothing if called. It is even more complicated in that environments named like that are some sort of `magic` tox environments and if the python string is present tox selects the corresponding python version. We might be able to create such environment that does all tests as described here but currently its not included and im not sure if one can simply override the magic env